cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2093,7 +2099,7 @@ int getPodIndexFromPvcName(String pvcName)  {
                     List<String> maybeDeletePvcs = res.result().stream().map(pvc -> pvc.getMetadata().getName()).collect(Collectors.toList());
                     List<String> desiredPvcs = zkCluster.generatePersistentVolumeClaims().stream().map(pvc -> pvc.getMetadata().getName()).collect(Collectors.toList());
 
-                    persistentClaimDeletion(maybeDeletePvcs, desiredPvcs).setHandler(futureResult.completer());
+                    persistentClaimDeletion(maybeDeletePvcs, desiredPvcs).setHandler(futureResult);
```

Comment:
Why are you changing this? Out of curiosity, if it works I'm fine with the change.

Reply:
`.completer()` is now deprecated. A `Future` is it's own completer (i.e. implements `Handler<AsyncResult<T>>`).

Reply:
Thanks